build/scripts/build.ps1

```diff
@@ -287,9 +305,13 @@ function Test-PerfRun() {
 }
 
 function Test-XUnitCoreClr() { 
-
     $unitDir = Join-Path $configDir "UnitTests"
-    $runtimeIdentifier = "win7-x64"
+    if (Is-Unix) {
+        $runtimeIdentifier = "ubuntu.14.04-x64"
```

Comment:
Advice on how to not hard-code this to Ubuntu 14 would be appreciated. The [current solution](https://github.com/dotnet/roslyn/blob/0867cd491fd3a3758e42d13d5e9b23d66986c6ff/build/scripts/tests.sh#L18) does a dynamic check by reading `dotnet --info` (I could replicate that in powershell if that's the best way to resolve this)

Reply:
Are we wanting to use the same RID as the current machine, the same RID as the current dotnet exe, or something else?

If possible (I don't remember if it works on nc1.1 apps), using `linux-x64` would probably be better.

Reply:
I think the issue with using `linux-x64` is that all of Roslyn doesn't use it yet. https://github.com/dotnet/roslyn/blob/a005d6883c5939705e3e9c41a80e18cece85eb20/build/Targets/Settings.props#L122

(netcoreapp1.1 does not have the `linux` RID at all, only 2.0 does, so it's a very new thing)

Reply:
> The dotnet exe RID should be identical to the current machine RID, and we need the RID to be able to run dotnet publish then immediately execute the outputs (with xunit).

I wasn't sure if this was the case for the new SDK builds where they target `linux-x64` as the RID

> I think the issue with using linux-x64 is that all of Roslyn doesn't use it yet. 

Yeah, although we may move to it when @jaredpar moves us to NC2.0 eventually

I don't know of a better way than running `dotnet --info`, but you could always ping the dotnet sdk team to make sure.